docs: Update ecommerce docs (#2327)
openedx/edx-documentation: en_us/install_operations/source/ecommerce-solutions.rst

.. _Adding ECommerce to Open edX:

Adding ECommerce to Open edX
#############################

This section is intended for those are who are building Open edX ecommerce
solutions and customizing an Open edX installation to support their use.

.. contents::
   :local:
   :depth: 1

> WooCommerce Plugin
> *******************
> 
> There exists a WooCommerce plugin for the Open edX platform. See the following resources:

Reviewer: It is a WordPress plugin that integrates WooCommerce with the platform.

```suggestion
Open edX Commerce (WordPress Plugin)
************************************

The Open edX WooCommerce Plugin is a free, open-source WordPress plugin that aims to integrate third-party e-commerce, WooCommerce, with your Open edX platform. See the following resources:
```

> * `WooCommerce Quickstart <https://edunext-docs-openedx-woocommerce-plugin.readthedocs-hosted.com/en/latest/plugin_quickstart.html#add-the-plugin-settings>`_

Reviewer: We can use the openedx doc, and it is the Open edX Commerce (WordPress Plugin) Quickstart.
```suggestion
* `Open edX Commerce (WordPress Plugin) Quickstart <https://docs.openedx.org/projects/wordpress-ecommerce-plugin/en/latest/plugin_quickstart.html>`_
```

> * `WooCommerce plugin source <https://github.com/openedx/openedx-wordpress-ecommerce?tab=readme-ov-file#-open-edx-commerce-wordpress-plugin>`_

Reviewer: ```suggestion
* `Open edX Commerce (WordPress Plugin) Source <https://github.com/openedx/openedx-wordpress-ecommerce?tab=readme-ov-file#-open-edx-commerce-wordpress-plugin>`_
```

Open edX Webhook Receiver
*************************

> The `Open edX Webhook Receiver <https://github.com/hastexo/webhook-receiver?tab=readme-ov-file#openedx-webhook-receiver>`

Reviewer: ```suggestion
The `Open edX Webhook Receiver <https://github.com/hastexo/webhook-receiver?tab=readme-ov-file#openedx-webhook-receiver>`_
```
